Two New Topics (#998)
osisoft/OCS-Docs: content/add-organize-data/organize-data/assets/remove-assets.md

---
uid: RemoveAssets
---

# Remove assets

Follow this procedure to remove assets from OCS. 

To remove a single asset:

1. From the left pane, select **Visualization** > **Asset Explorer**.

> 1. From the **Assets/Asset** Types dropdown list, select **Assets**.

Reviewer: ```suggestion
1. From the **Assets/Asset Types** selector, select **Assets**.
```

> 1. Select the checkbox for the Asset you want to remove.

Reviewer: ```suggestion
1. Select the checkbox for the asset you want to remove.
```

> 1. Select **More Options**, then select **Remove**.

Reviewer: ```suggestion
1. Select **More options** ![More options icon](../../../_icons/default/dots-vertical.svg) in the `Asset Details` pane, then select **Remove**.
```

1. At the prompt, select **Remove**. 

> To remove a multiple assets:

Reviewer: ```suggestion
To remove multiple assets:
```

1. From the left pane, select **Visualization** > **Asset Explorer**.

> 1. From the **Assets/Asset** Types dropdown list, select **Asset Types**.

Reviewer: ```suggestion
1. From the **Assets/Asset Types** selector, select **Assets**.
```

> 1. Select the checkboxes for the Assets you want to remove.

Reviewer: ```suggestion
1. Select the checkboxes for the assets you want to remove.
```

1. Select **Remove**.

1. At the prompt, select **Remove**.

> **Note:** You may receive an error message when attempting to delete asset types. Only the creator of the asset type or a user with access can delete an asset type.

Reviewer: ```suggestion
**Note:** You may receive an error message when attempting to delete assets. Only the creator of the asset or a user with access can delete an asset.
```
